mender-dist-packages-building/docker-build-image

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
```

Comment:
nit: we never add a new line here.
q: do we have a license and format checks in here for the shell code?

Reply:
I copied the script as-is in the original repository. I would like to not modify it during this task.